### ISSUE TITLE
meta-mender layer causes onie image not to boot

### ISSUE THREAD

**Issue by johntoomey** [NetworkGradeLinux/mion#37]

**Comment by johntoomey** [comment]
Latest code in the mender dunfell branch appears to fix this issue but causes the mender image to fail to build - after talking to the mender folks they have a fix for this issue in the following PR: https://github.com/mendersoftware/meta-mender/pull/1167

we need to wait to see if they can get this patch into the dunfell branch fairly quickly or if we have to some up with some sort of creative workaround.

**Comment by johntoomey** [comment]
New fixes provided by the mender folks seem to work - just waiting for them to merge these into the dunfell branch before we can close this issue out

https://github.com/mendersoftware/meta-mender/pull/1175

**Comment by johntoomey** [comment]
Fix has bee merged into meta-mender master branch https://github.com/mendersoftware/meta-mender/pull/1175

Its not clear if this is going to be cherry-picked into the dunfell branch

**Comment by johntoomey** [comment]
Contacted the mender folks and asked how to get the two commits that fix the issue cherry-picked onto the dunfell branch.

**Comment by johntoomey** [comment]
Tried backporting the fixes myself but got lots of merge conflicts - requested backport on the mender hub https://hub.mender.io/t/issue-bulding-mender-dunfell-branch-backport-request/2940
